opus/application/static_media/js/browse.js

```diff
@@ -1440,6 +1435,54 @@ var o_browse = {
         o_browse.tableScrollbar.update();
     },
 
+    adjustmetadataModalMenu: function() {
```

Comment:
What is a metadataModalMenu? It's not obvious to me what menu we're talking about.

Reply:
Rename it to adjustMetadataSelectorMenu